packages/node/src/BacktraceClient.ts

```diff
@@ -25,9 +27,82 @@ export class BacktraceClient extends BacktraceCoreClient {
             undefined,
             new VariableDebugIdMapProvider(global as DebugIdContainer),
         );
+        if (options.captureUnhandledErrors !== false) {
+            this.captureUnhandledErrors();
+        }
```

Comment:
We should always capture unhandled exceptions, but not unhandled rejections.

Either add two options for unhandled exceptions and rejections, or change this option to control only unhandled rejections.

Reply:
Adjusted the code to our requirements.